smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftSettings.kt

```diff
@@ -82,8 +85,21 @@ class SwiftSettings(
             val swiftVersion = config.expectStringMember(SWIFT_VERSION).value
             val sdkId = sanitizeSdkId(config.getStringMemberOrDefault(SDK_ID, serviceId.name))
             val useInterceptors = config.getBooleanMemberOrDefault(USE_INTERCEPTORS)
-
-            return SwiftSettings(serviceId, moduleName, version, desc, author, homepage, sdkId, gitRepo, swiftVersion, useInterceptors)
+            val mergeModels = config.getBooleanMemberOrDefault(MERGE_MODELS)
```

Comment:
This sets the `SwiftSettings` field based on the contents of `smithy-build.json`.